libmachine/host.go

```diff
@@ -167,7 +155,7 @@ func (h *Host) Start() error {
 		return err
 	}
 
-	return utils.WaitFor(h.MachineInState(state.Running))
+	return utils.WaitFor(drivers.MachineInState(h.Driver, state.Running))
```

Comment:
This is starting to get clunky.  I'm not saying it's bad - it just seems getting a bit much with the nesting.  Not sure what to do.